autodh/_version.py

```diff
@@ -0,0 +1 @@
+__version__ = '0.0.1'
```

Comment:
something got messed up in the versioneer installation. this file shouldn't be hardcoded to a specific version, the `__init__.py` is missing an import of this file and there is no `MANIFEST.in` file. try to run `versioneer install` in the root of the repo to fix it.

Reply:
`(venv3.6) root@4fd0a2831112:~/AutoDH# versioneer install
Traceback (most recent call last):
  File "/root/venv3.6/bin/versioneer", line 5, in <module>
    from versioneer import main
ImportError: cannot import name 'main'`

unsure whats happening. At first it couldn't find versioneer so i pip installed it- unfortunately it led to this error.